iib/workers/tasks/utils.py

```diff
@@ -319,7 +319,12 @@ def skopeo_inspect(*args, return_json=True):
     cmd = ['skopeo', '--command-timeout', skopeo_timeout, 'inspect'] + list(args)
     output = run_cmd(cmd, exc_msg=exc_msg)
     if return_json:
-        return json.loads(output)
+        output_json = json.loads(output)
+        if output_json.get('mediaType') == None:
```

Comment:
I think you can get mediaType only if you use `--raw` otherwise you can find it. So one option is to check that skopeo is called with `--raw` or call it again with --raw if it is not called correctly

Reply:
the problem is not here but probably where it was reported https://github.com/release-engineering/iib/blob/master/iib/workers/tasks/utils.py#L89.

Reply:
@lipoja @pbortlov 

`mediaType` will be in the result when passing `--raw` flag in the `skopeo inspect` command, and  it's passed to the command in the `get_resolved_bundles` method. Anyway, I made some modifications as I realized I overlooked some details.